Flutter_Weather/lib/widgets/weather_info_widget.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_weather/packages/dafluta/dafluta.dart';
+import 'package:flutter_weather/resources/palette.dart';
+import 'package:weather_icons/weather_icons.dart';
+
+class WeatherInfo extends StatelessWidget {
+  final String weatherType;
+  final double upperLimitTemp;
+  final double lowerLimitTemp;
+  final double currentTemp;
+
+  const WeatherInfo({
+    Key? key,
+    required this.weatherType,
+    required this.upperLimitTemp,
+    required this.lowerLimitTemp,
+    required this.currentTemp
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.all(15),
+      child: Column(
+        children: [
+          VBox(MediaQuery.of(context).size.height * 0.65),
+          WeatherType(weatherType),
+          VBox(15),
+          TempLimit(upperLimitTemp, lowerLimitTemp),
+          VBox(15),
+          CurrentTemp(currentTemp)
+        ],
+      ),
+    );
+  }
+}
+
+Row WeatherType(String weatherType) {
```

Comment:
Should not use global functions in object oriented programming. Please move these functions inside a widget.